_studio/mfx_lib/shared/src/libmfxsw_encode.cpp

```diff
@@ -293,6 +311,11 @@ class FallbackProxyENCODE : public VideoENCODE
     {
         assert(m_impl);
         mfxStatus mfxRes = m_impl->Reset(par);
+        // SW fallback if EncodeGUID is absence
```

Comment:
What does it mean 'EncodeGUID is absence' in that context? I see no 'EncodeGUID' around, comment is just confusing.

Reply:
There are 4 places with that, I have nothing to clarify, so we may want to just drop them.